format/apple/bplist/ns_keyed_archiver.jq

```diff
@@ -0,0 +1,70 @@
+def from_ns_keyed_archiver:
+  (  . as {
```

Comment:
Tiny left indent

Reply:
I think I fixed it, hopefully at some point can figure out some kind of formatting solution for neovim.

Reply:
Aha that might explain things, vscode-jq does a quite ok job based on the highlighting syntax it seems... but is not great.